docs/source/api.rst

```diff
@@ -0,0 +1,11 @@
+pyspool API Documentation
+=========================
+
+Spool
+-----
+.. automodule:: spool
+
+.. autoclass:: Spool
+    :members:
+
+.. :members: __init__, register_piece, register, editions, consigned_registration, transfer, consign, unconsign, loan, migrate, refill_main_wallet, refill, simple_spool_transaction, select_inputs
```

Comment:
It is in the `docstrings` already. https://github.com/sbellem/pyspool/blob/fc995ae8f6b8f82ec7a13b2387bef804eb505550/spool/spool.py#L24-L49

:smile: